packages/runtime/src/store.ts

```diff
@@ -3,43 +3,85 @@ import dayjs from 'dayjs';
 import relativeTime from 'dayjs/plugin/relativeTime';
 import { reactive } from '@vue/reactivity';
 import { watch } from '@vue-reactivity/watch';
+import { LIST_ITEM_EXP, LIST_ITEM_INDEX_EXP } from './constants';
 
 dayjs.extend(relativeTime);
 
-function parseExpression(raw: string): {
-  dynamic: boolean;
+type ExpChunk = {
   expression: string;
-} {
-  if (!raw) {
-    return {
-      dynamic: false,
-      expression: raw,
-    };
-  }
-  const matchArr = raw.match(/{{(.+)}}/);
-  if (!matchArr) {
-    return {
-      dynamic: false,
-      expression: raw,
-    };
-  }
-  return {
-    dynamic: true,
-    expression: matchArr[1],
-  };
-}
+  isDynamic: boolean;
+};
+
+// TODO: use web worker
+const builtIn = {
+  dayjs,
+};
 
 function isNumeric(x: string | number) {
-  return !isNaN(Number(x));
+  return !isNaN(Number(x)) && x !== '';
 }
 
 export const stateStore = reactive<Record<string, any>>({});
 
-// TODO: use web worker
-const builtIn = {
-  dayjs,
-};
-function maskedEval(raw: string) {
+function parseExpression(str: string, parseListItem = false): ExpChunk[] {
```

Comment:
I think we need some unit tests for this function

Reply:
OK, I have added unit test